src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public class FlywheelCommands {
+    private Flywheel flywheel;
+
+    public FlywheelCommands(Flywheel flywheel) {
+        this.flywheel = flywheel;
+    }
+
+    public Command setPowers(double rightPower, double leftPower){
+        return new FunctionalCommand(
+                () -> {},
+                () -> flywheel.setPowers(rightPower, leftPower),
+                (interrupted) -> flywheel.stop(),
+                () -> false,
+                flywheel
+        );
+    }
+
+    public Command setVelocities(Rotation2d rightVelocity, Rotation2d leftVelocity) {
+        return new FunctionalCommand(
+                () -> {
+                },
+                () -> flywheel.setVelocities(rightVelocity, leftVelocity),
+                (interrupted) -> flywheel.stop(),
+                () -> false,
+                flywheel
```

Comment:
add is at velocities maybe, and dont stop (i think it will be easier to use in sequnces)